src/stream.c

```diff
@@ -505,7 +542,7 @@ int stream_alloc(struct stream **sp, struct list *streaml,
 	if (cfg->jbuf_del.min && cfg->jbuf_del.max) {
 
 		err = jbuf_alloc(&s->jbuf, cfg->jbuf_del.min,
-				 cfg->jbuf_del.max);
+				 cfg->jbuf_del.max, cfg->jbuf_wish);
```

Comment:
perhaps avoid breaking the api by adding a new function, e.g.:

```c
void jbuf_set_wish(struct jbuf *jb, uint32_t wish);
```

Reply:
I'll add a commit to avoid the push force. Maybe later we should make a interactive rebase before merge.